config/dev.json

```diff
@@ -95,5 +95,6 @@
   "openid": {
     "keyFile": "../config/key.json",
     "key": {}
-  }
+  },
+  "fxaEnv": "development"
```

Comment:
`env` is node environment, fxaEnv is Firefox Accounts running mode. 

We discussed adding this in the future to stop relying on node's env. @jrgm thoughts?